PlayerControls/sources/AdVideoControls.swift

```diff
@@ -64,25 +67,49 @@ public final class AdVideoControls: UIViewController {
         seekerView.accessibilityLabel = props.seeker?.accessibilityLabel ?? ""
         remainingPlayTimeLabel.text = props.seeker?.remainingPlayTime
         airplayActiveView.isHidden = props.airplayActiveViewHidden
+        
+        adSkipButton.titleLabel?.textAlignment = .center
+        
+        switch props.adSkipState {
+        case .unavailable:
+            adSkipButton.isHidden = true
+        case .available:
+            if adSkipButton.isEnabled == false {
```

Comment:
Why do we need this `if`

Reply:
Yeah, just make `isEnabled = true` also.

Reply:
Dropped as it is not needed anymore